WordPress/src/main/java/org/wordpress/android/datasets/PeopleTable.java

```diff
@@ -80,11 +83,28 @@ public static void savePeople(List<Person> peopleList, int localTableBlogId) {
         }
     }
 
+    public static int getPeopleCountForLocalBlogId(int localTableBlogId) {
+        String[] args = new String[]{Integer.toString(localTableBlogId)};
+        String sql = "SELECT COUNT(*) FROM " + PEOPLE_TABLE + " WHERE local_blog_id=?";
+        return SqlUtils.intForQuery(getReadableDb(), sql, args);
+    }
+
     public static void deletePeopleForLocalBlogId(int localTableBlogId) {
         String[] args = new String[]{Integer.toString(localTableBlogId)};
         getWritableDb().delete(PEOPLE_TABLE, "local_blog_id=?", args);
     }
 
+    public static void deletePeopleForLocalBlogIdExceptForFirstPage(int localTableBlogId) {
+        int size = getPeopleCountForLocalBlogId(localTableBlogId);
+        int fetchLimit = PeopleUtils.FETCH_USERS_LIMIT;
+        if (size > fetchLimit) {
+            int deleteCount = size - fetchLimit;
+            String[] args = new String[]{Integer.toString(deleteCount), Integer.toString(localTableBlogId)};
+            getWritableDb().delete(PEOPLE_TABLE, "person_id " +
```

Comment:
I've added a question to the PR description just for this 👊, because I was trying to use `?` and pass the `PEOPLE_TABLE` as an argument. Working on that sql statement probably fried my brain a bit. 😢